src/CanvasView.tsx

```diff
@@ -144,42 +141,50 @@ export const CanvasView: React.FC = () => {
             RESET
           </Button>
         )}
-        <Menu closeOnSelect={true} placement="top-end">
-          <MenuButton
-            as={IconButton}
-            size="sm"
-            isRound
-            aria-label="More info"
-            marginLeft="auto"
-            variant="secondary"
-            icon={
-              <QuestionOutlineIcon
-                boxSize={6}
-                css={{ '& circle': { display: 'none' } }}
-              />
-            }
-          />
-          <Portal>
-            <MenuList fontSize="sm" padding="0">
-              <MenuItem
-                as={Link}
-                href="https://github.com/statelyai/xstate"
-                target="_blank"
-                rel="noreferrer"
-              >
-                XState version 4.23.0
-              </MenuItem>
-              <MenuItem
-                as={Link}
-                href="https://stately.ai/privacy"
-                target="_blank"
-                rel="noreferrer"
-              >
-                Privacy Policy
-              </MenuItem>
-            </MenuList>
-          </Portal>
-        </Menu>
+        <ButtonGroup size="sm" spacing={2} marginLeft="auto">
+          <Menu closeOnSelect={true} placement="top-end">
+            <MenuButton
+              as={IconButton}
+              size="sm"
+              isRound
+              aria-label="More info"
+              variant="secondary"
+              icon={
+                <QuestionOutlineIcon
+                  boxSize={6}
+                  css={{ '& circle': { display: 'none' } }}
+                />
+              }
+            />
+            <Portal>
+              <MenuList fontSize="sm" padding="0">
+                <MenuItem
+                  as={Link}
+                  href="https://github.com/statelyai/xstate"
+                  target="_blank"
+                  rel="noreferrer"
+                >
+                  XState version 4.23.0
+                </MenuItem>
+                <MenuItem
+                  as={Link}
+                  href="https://stately.ai/privacy"
+                  target="_blank"
+                  rel="noreferrer"
+                >
+                  Privacy Policy
+                </MenuItem>
+              </MenuList>
+            </Portal>
+          </Menu>
+          <IconButton
+            aria-label="Collapse panels"
```

Comment:
This should be conditional, like the `icon` prop, as well. The aria-label should describe the currently available action accurately.